packages/core/package.json

```diff
@@ -33,6 +33,7 @@
     "babel-plugin-auto-import": "^1.0.5",
     "babel-plugin-graphql-tag": "^2.5.0",
     "babel-plugin-module-resolver": "^4.0.0",
+    "babel-plugin-tester": "^9.0.1",
```

Comment:
I think this should be added to the devDeps instead of deps (otherwise it's installed in every Redwood app.)

Reply:
Ah, you are right! Missed the `-D` there.